### PR TITLE
REST > Charging station QRCode generate/download + getOcppParameters schemas

### PR DESCRIPTION
**Pull request by Ninjeneer**

### CODE REVIEW

File: src/assets/server/rest/v1/schemas/chargingstation/chargingstation-qrcode-download.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    },
+    "SiteAreaID": {
+      "$ref": "common.json#/definitions/ids"
+    },
+    "SiteID": {
+      "$ref": "common.json#/definitions/ids"
+    },
+    "ProjectFields": {
+      "$ref": "common.json#/definitions/projectFields"
+    }
```
Comment (LucasBrazi06) [inline src/assets/server/rest/v1/schemas/chargingstation/chargingstation-qrcode-download.json:19]: I think there is no projected fields in downloading the QR-Code as you get a PDF

File: src/assets/server/rest/v1/schemas/chargingstation/chargingstation-qrcode-generate.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  "title": "Generate QRCode for Charging Station",
+  "type": "object",
+  "properties": {
+    "ChargingStationID": {
+      "$ref": "chargingstation.json#/definitions/id"
+    },
+    "ConnectorID": {
+      "$ref": "chargingstation.json#/definitions/connectorID"
+    },
+    "ProjectFields": {
```
Comment (LucasBrazi06) [inline src/assets/server/rest/v1/schemas/chargingstation/chargingstation-qrcode-generate.json:11]: I think there is no projected fields in downloading the QR-Code as you get a PDF

File: src/assets/server/rest/v1/schemas/chargingstation/chargingstation.json [hunk 3]
```diff
@@ -91,6 +91,11 @@
       "type": "number",
       "sanitize": "mongo"
     },
+    "connectorIDs": {
+      "type": "string",
```
Comment (LucasBrazi06) [inline src/assets/server/rest/v1/schemas/chargingstation/chargingstation.json:95]: Isn't is an array on Numbers?

**Review by LucasBrazi06 — CHANGES_REQUESTED** [review]
Check my comments

File: src/assets/server/rest/v1/schemas/chargingstation/chargingstation-qrcode-download.json (hunk 1 above)
Comment (Ninjeneer) [reply src/assets/server/rest/v1/schemas/chargingstation/chargingstation-qrcode-download.json:19]: Indeed, removed

File: src/assets/server/rest/v1/schemas/chargingstation/chargingstation-qrcode-generate.json (hunk 2 above)
Comment (Ninjeneer) [reply src/assets/server/rest/v1/schemas/chargingstation/chargingstation-qrcode-generate.json:11]: Indeed, removed

File: src/assets/server/rest/v1/schemas/chargingstation/chargingstation.json (hunk 3 above)
Comment (Ninjeneer) [reply src/assets/server/rest/v1/schemas/chargingstation/chargingstation.json:95]: This is the user side input, so we receive a pipe separated string here, splited and converted on service side

**Review by LucasBrazi06 — APPROVED** [review]

**Pull request merged**